spec/models/spreadsheets/primary_school_spreadsheet_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Spreadsheets::PrimarySchoolSpreadsheet do
+  describe "#process" do
+    let!(:ps_spreadsheet) { Spreadsheets::PrimarySchoolSpreadsheet.new }
+
+    context "row with full info" do
+      let(:row) {{ "Province" => "Koh Kong", "District" => "Botum Sakor", "Commune" => "Andoung Tuek", "commune_code" => "090101", "school_code" => "090101_1", "school_name_en" => "Andaung Teuk", "school_name_km" => "អណ្តូងទឹក" }}
```

Comment:
@hengsokly please remove not relevant fields province, district, commune and keep only commune_code so that we simply stub only the need of attributes